main.py

```diff
@@ -1,10 +1,43 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as config:
+        players = json.loads(config.read())
+
+    for nickname, params in players.items():
+        params_race = params["race"]
+        race, _ = Race.objects.get_or_create(
+            name=params_race["name"],
+            description=params_race["description"],
+        )
+
+        params_skills = params_race["skills"]
+        for skill in params_skills:
```

Comment:
Directly iterate over `params_race["skills"]`. No need to have this `params_skills` variable